lib/client.js

```diff
@@ -182,17 +194,15 @@ var GiantSwarm = (function () {
       if (typeof successCallback !== 'function') throw new this.UserException("Parameter successCallback must be of type function");
       if (errorCallback !== null && typeof errorCallback !== 'function') throw new this.UserException('errorCallback must be null or function');
       params = params || '';
-      var headers = {
-        'Authorization': 'giantswarm ' + authToken
-      }
-      if (clusterId) {
-        headers[CLUSTER_ID_HEADER] = clusterId;
-      }
+      var headers = { 'Authorization': 'giantswarm ' + authToken }
```

Comment:
I feel like spreading this out on multiple lines doesn't help my readability.
The single line is more readable to me. What do you think?